ako-operator/bundle/manifests/ako-operator.clusterserviceversion.yaml

```diff
@@ -586,25 +625,27 @@ spec:
     supported: false
   relatedImages:
   - name: ako-operator
-    image: projects.registry.vmware.com/ako/ako-operator@sha256:629c50ebe19c035d292ff5a3c45ccb4482f4b839234891f54087e6e4654b26ee
+    image: projects.registry.vmware.com/ako/ako-operator@sha256:ed71b04ad93ef35920329662e9fecedec9079243dfa0561b62e05d618f6dce6f
   - name: ako
-    image: projects.registry.vmware.com/ako/ako@sha256:7c4f8c84eb5b1216f917ad1183c7e9ae19cd9b58d2a9669d7fb24b70156fc7e3
+    image: projects.registry.vmware.com/ako/ako@sha256:403e629eeb194a56cc5f71c29363df5c94ebb6afe0cb823120ce23dc701fe1de
+  - name: ako-gateway-api
+    image: projects.registry.vmware.com/ako/ako-gateway-api@sha256:089e4ccd0642954cc031eba541b0e3c84b5d88c1c96ae5f788edeed30ecfd26d
   keywords:
```

Comment:
As we are introducing GatewayAPI support also. We can add that keyword also.

Reply:
Added the GatewayAPI keyword.